app/assets/javascripts/components/story/ExpandedStory/ExpandedStorySection.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const ExpandedStorySection = ({ title, children, content }) =>
```

Comment:
I think `content` is a misleading name here because it looks like it's the actual content of the section but it's not, the content is given by the `children` prop. What do you think about renaming this prop to `identifier`?